extension/dynamicconfig/model/config_block.go

```diff
@@ -0,0 +1,40 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package model
+
+type ConfigBlock struct {
+	Resource     []string
```

Comment:
I thought Resource is a list of label key/value pairs. Here it is just an array of string. So these are label values and the label keys are implicit?

Reply:
To simplify specifying key-value pairs within the constraints of yaml, I decided that users can specify them as a string like `"key: value"`, which would match a resource that has the corresponding keys and values. In other words, matches are computed on a normalized string version of the key-value pairs (see for example the `attrToString` function).

The alternative would be specifying key-value pairs explicitly in the yaml, maybe something like:
```
Resource:
    - Key: namespace
      Value: observability
    - Key: deployment-id
      Value: 20201130 
``` 

Though it's a little more verbose. What are your thoughts between these two methods?

Reply:
Sounds good.